### ISSUE TITLE
Reduce font size of ToC content

### ISSUE THREAD

**Issue by j-low** [cockroachdb/docs#2370]
__[description]__
* Previous work had increased the font size of ToC content to address some accessibility concerns with the previously very small font.  However, the font size increase appears a bit too large and should be reduced slightly to match the font size of the side nav list items.

**Comment by j-low** [comment]
Address in PR #2371.  Closing issue.
